rte/mo_rte_lw.F90

```diff
@@ -214,12 +229,23 @@ function rte_lw(optical_props, top_at_1, &
         !$acc enter data copyin(optical_props%tau)
         error_msg =  optical_props%validate()
         if(len_trim(error_msg) > 0) return
-        call lw_solver_noscat_GaussQuad(ncol, nlay, ngpt, logical(top_at_1, wl), &
-                              n_quad_angs, gauss_Ds(1:n_quad_angs,n_quad_angs), gauss_wts(1:n_quad_angs,n_quad_angs), &
+
+        if (present(lw_Ds)) then
+          call lw_solver_noscat(ncol, nlay, ngpt, &
+                                logical(top_at_1, wl), lw_Ds, gauss_wts(1,1), &
```

Comment:
As an aesthetic choice, can the layout of the lines in the two branches here be the same, i.e. logical(top_at_1, wl) sits on its own line in both branches?